src/toTransform.js

```diff
@@ -35,14 +35,45 @@
     return 'scale3d(' + valuesArray.join(', ') + ')';
   }
 
+  var pathElement = document.createElementNS('http://www.w3.org/2000/svg', 'path');
+
+  function convertPath (properties) {
+    var offsetPath = internalScope.offsetPathParse(properties['offset-path']);
+    if (!offsetPath || offsetPath.type !== 'path') {
+      return null;
+    }
+
+    var offsetDistance;
+    if (properties['offset-distance'] === undefined) {
+      offsetDistance = {value: 0, unit: 'px'};
```

Comment:
We want to use this default value if the parse below fails. Add test for this scenario.